proxy-config.json

```diff
@@ -0,0 +1,103 @@
+{
```

Comment:
Maybe you can describe this file a bit and how we use it, how others can cofigure it ?